script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1D.txt

```diff
@@ -9,10 +9,10 @@ I look up and see that most of the students in the classroom are spending their
 <Page3>
 The teachers who are over thirty say that today's students seem to have no friends in real life.
 Of course, that's an outdated misconception and one that seems unique to people from the previous generation.
-Dependence on the internet and dependence on friends in front of you have become equivalent, and no one in our class is at odds over that.
+Dependence on the internet and dependence on friends in front of you have become equivalent, and no one in our class is at odds over it.
 <Page4>
 School used to be a place where you could only make friends inside the classroom. There was little connection to the outside world.
-It's different now. You can find as many friends as you want. The atmosphere in the classroom doesn't have to be homey. As long as it's not unpleasant, then it's fine.
+It's different now. You can find as many friends as you want. The atmosphere in the classroom doesn't have to be homey. As long as it's not unpleasant, it's fine.
```

Comment:
Might prefer to keep `then` here since there's a clear cause -> effect relation he's emphasizing

Reply:
I'm for removing 'then'